feat: Add @DChitale as a contributor

DChitale opened the pull request:
<!-- Please fill in all areas in this PR form. Incomplete PRs will be marked invalid and may be closed. -->

## Description

<!--
Please do not leave this blank
This PR adds <github-username> as a contributor.
-->

## What type of PR is this? (check all applicable)

- [x] 🤝 Add a contributor
-  [x] 📝 Documentation Update

## Related Issues

<!-- 
Please use this format link issue numbers: Closes #123
More information: https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword 
-->

## Contributors Checklist

### I've read through the [Getting Started](https://intro.opensauced.pizza/#/intro-to-oss/how-to-contribute-to-open-source?id=getting-started) section.

- [x] ✅ Yes
- [ ] ❌ Not yet

### Have you run `npm run contributors:generate` to generate your profile and the badge on the README?

- [x] ✅ Yes
- [ ] ❌ No

## Added to documentation?

- [x] 📜 README.md
- [ ] 🙅 no documentation needed

## Screenshot (Required for PR Review)

![image](https://github.com/open-sauced/guestbook/assets/142161975/c2ce04cf-f73a-40ed-8599-4a0da379f9b6)

<!-- Please provide a screenshot of your profile being generated on the README. This ensures that you ran the `npm run contributors:generate` command, as mentioned in the previous question, which makes it easier for the maintainers to review PRs. All PRs without screenshots will be automatically rejected-->

## [optional] What GIF best describes this PR or how it makes you feel?

<!-- note: PRs with deleted sections will be marked invalid -->

<!--
  For Work In Progress Pull Requests, please use the Draft PR feature,
  see https://github.blog/2019-02-14-introducing-draft-pull-requests/ for further details.

  For a timely review/response, please avoid force-pushing additional
  commits if your PR already received reviews or comments.

  Before submitting a Pull Request, please ensure you've done the following:
  - 📖 Read the Open Sauced Contributing Guide: https://github.com/open-sauced/.github/blob/main/CONTRIBUTING.md.
  - 📖 Read the Open Sauced Code of Conduct: https://github.com/open-sauced/.github/blob/main/CODE_OF_CONDUCT.md.
  - 👷‍♀️ Create small PRs. In most cases, this will be possible.
  - 📝 Use descriptive commit messages.
  - 📗 Update any related documentation and include any relevant screenshots.
-->

adiati98 reviewed (changes requested):
Hey @DChitale,
Thank you for your contribution! ✨ 

We have merged a PR before yours. Please **[resolve the merge conflicts](https://intro.opensauced.pizza/#/intro-to-oss/how-to-contribute-to-open-source?id=merge-conflicts-in-the-guestbook-repository)** before we can review your PR.

Do let us know if you have questions or need any help. Thanks. 😊

DChitale commented:
Can you assist me how can i do that just by deleting the middle lines?
![image](https://github.com/open-sauced/guestbook/assets/142161975/d4f4dffc-7825-4062-a76e-f4c58f2754b0)

adiati98 commented on kla:
Please remove this file as this is not part of the changes.

adiati98 commented:
Hey @DChitale,

From the screenshot, it seems that you need to:

1. Add a closing bracket followed by a comma ( `},`) to close the data of the contributor before yours.
2. Wrap your data inside an opening and closing brackets (`{}`).

Are you solving this directly on GitHub or VS Code?

I recommend you solve the conflicts on VS Code because you will need to run the command to (re)generate your profile after resolving the conflicts.

DChitale commented:
@adiati98 I think i have fixed the conflict please review it 
![image](https://github.com/open-sauced/guestbook/assets/142161975/ce94bbd0-a68b-47ff-aa7e-6f4400ae6a94)

adiati98 commented on oj:
This file shouldn't be here as well. Please remove it.

adiati98 reviewed (changes requested):
Hey @DChitale,

There are a few things below. I created checklists to make it easier for you. Check each one once you complete a task:

- [ ] Please delete the file that I've commented on.
- [ ] There is a failed check that you can [read here](https://github.com/open-sauced/guestbook/pull/336#pullrequestreview-2080804495). Read the bot message and please address it until all checks passed.
- [ ] Please complete the PR template. That said, you still need to add a description in your form. We won't merge PR that's incomplete.

adiati98 commented:
Hi @DChitale,

We have merged a PR before yours. Please do the following:

- [ ] Pull the latest changes to your local branch. You can do so by running the command `git pull`.
- [ ] Run the `run npm contributors:generate` command in your terminal.
- [ ] Push the changes to your remote branch by running `git push`.
- [ ] Address my [previous comment](https://github.com/open-sauced/guestbook/pull/336#pullrequestreview-2081283126) before we can review your PR.

Please let me know if you have any questions or need help. Thanks! ✨

DChitale commented:
@adiati98 I have done all tasks that you mentioned the PR template is also added the issue address in it

adiati98 commented:
@DChitale, if you check each checklist that I provided for you after you finish fixing each one of them, you'll see that you haven't addressed some of them. 🙂

Please do so, and let me know once you're done addressing all of them. Thank you ✨️

DChitale commented:
@adiati98 I think i done all the things that you mentioned 
deleted the file  (oj file). 
 There is a failed check that you can https://github.com/open-sauced/guestbook/pull/336#pullrequestreview-2080804495. 

![image](https://github.com/open-sauced/guestbook/assets/142161975/63abb6ec-96d2-4866-bb04-af6d7a1e6ff4)

Added description also 

please tell me if i missed anything thank you

adiati98 reviewed (approved):
Well done and thank you for the fixes, @DChitale! ✨ 

If you haven't, you can join our community [Discord](https://discord.com/invite/U2peSNf23P).
Also, check out our [pizza-verse](https://github.com/open-sauced/pizza-verse) repo if you want to contribute further! 🍕🍴

![git merge GIF](https://media.giphy.com/media/cFkiFMDg3iFoI/giphy.gif)
